Services/ValidationService.js

```diff
@@ -1 +1,21 @@
- 
\ No newline at end of file
+let validateVisitorSignIn = (visitorInput) => {
+
+    return (checkInputProvided(visitorInput) && checkLength(visitorInput.forename) && checkLength(visitorInput.surname) && checkMaxLength(visitorInput.company));
```

Comment:
```suggestion
    return (checkInputProvided(visitorInput) && checkLength(visitorInput.forename)
    && checkLength(visitorInput.surname) && checkMaxLength(visitorInput.company));
```
would press Enter halfway along as shown here to make the line more readable

Reply:
Done :)